server/command.go

```diff
@@ -227,28 +228,7 @@ func (p *Plugin) handleSubscribe(_ *plugin.Context, args *model.CommandArgs, par
 		}
 
 		if repo == "" {
-			if err = p.SubscribeOrg(ctx, bitbucketClient, args.UserId, owner, args.ChannelId, features); err != nil {
-				return err.Error()
-			}
-
-			orgLink := fmt.Sprintf("%s%s", p.getBaseURL(), owner)
-			msg := fmt.Sprintf("Successfully subscribed to organization [%s](%s) with events: %s", owner, orgLink, formattedString(features))
-			if previousSubscribedEvents != "" {
-				msg += fmt.Sprintf("\nThe previous subscription with: %s was overwritten.\n", formattedString(previousSubscribedEvents))
-			}
-
-			post := &model.Post{
-				ChannelId: args.ChannelId,
-				UserId:    p.BotUserID,
-				Message:   msg,
-			}
-
-			if _, appErr := p.API.CreatePost(post); appErr != nil {
-				p.API.LogWarn("error while creating post", "post", post, "error", appErr.Error())
-				return fmt.Sprintf("%s Though there was an error creating the public post: %s", msg, appErr.Error())
-			}
-
-			return msg
+			return requiredErrorMessage
```

Comment:
It looks like we can't subscribe to orgs anymore. Is this intentional?

Reply:
@mickmister yes can see the comment https://github.com/mattermost/mattermost-plugin-bitbucket/issues/73#issue-1175540416